PiwikTracker/Dispatcher.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+internal protocol Dispatcher {
+    associatedtype Dispatchable
+    
+    var userAgent: String? { get set }
```

Comment:
Maybe the protocol should only expose the userAgent as { get }
It is up the implementation how the value is create i.e.
* It uses the default value provided by the platform
* It receives the value as a parameter in the init call
But normally it should not be changed after its been assigned